research/quantifying-data-leakage/job_configs/cxr_9clients/client9/config/config_fed_client.json

```diff
@@ -1,7 +1,8 @@
 {
     "format_version": 2,
```

Comment:
since we are using `research/quantifying-data-leakage/src/nvflare_gradinv/utils/modify_client_config.py` to modify the client configs, do you think we could just have 1 version of "config_fed_client.json" and use that script to generate (modify and save) different configs for different client sites?

Reply:
Changed it to create job configs from a template for the two described experiments.